source/_components/climate.generic_thermostat.markdown

```diff
@@ -14,36 +14,85 @@ ha_iot_class: "Local Polling"
 ---
 
 
-The `generic_thermostat` climate platform is a thermostat implemented in Home Assistant. It uses a sensor and a switch connected to a heater or air conditioning under the hood. When in heater mode, if the measured temperature is cooler then the target temperature, the heater will be turned on and turned off when the required temperature is reached. When in air conditioning mode, if the measured temperature is hotter then the target temperature, the air conditioning will be turned on and turned off when required temperature is reached.
+The `generic_thermostat` climate platform is a thermostat implemented in Home Assistant. It uses a sensor and switches to control heaters and A/C units. In can be used for heating, cooling or both at the same time.
+
+## Heater mode
+You can choose to use `generic_thermostat` as heater only by applying a simple configuration as below:
 
 ```yaml
-# Example configuration.yaml entry
+# Heating example configuration.yaml entry
 climate:
   - platform: generic_thermostat
     name: Study
-    heater: switch.study_heater
+    heater_control: switch.study_heater
     target_sensor: sensor.study_temperature
+    min_cycle_duration:
+      seconds: 30
+```
+
+Note: `min_cycle_duration` is recommended to avoid triggering your ignition too often when doing changes (manual or through automation).
+
+## A/C mode
+You can choose to use `generic_thermostat` as A/C unit by using a configuration as below:
+
+```yaml
+# A/C example configuration.yaml entry
+climate:
+  - platform: generic_thermostat
+    name: Study
+    ac_control: switch.study_ac
+    target_sensor: sensor.study_temperature
+    keep_alive:
+      minutes: 3
 ```
 
-Configuration variables:
+Note: `keep_alive` is usually recommended as most A/C units will turn off after a while without a keep alive.
+
+## Auto - Using both a heater and an A/C unit together
+You can have both a heater and an A/C unit setup inside `generic_thermostat` to allow `auto` operation mode with a low target temperature and a high target temperature. 
+
+```yaml
+# Auto mode example configuration.yaml entry
+climate:
+  - platform: generic_thermostat
+    name: Study
+    ac_control: switch.study_ac
+    heater_control: switch.study_heater
+    target_sensor: sensor.study_temperature
+    target_temp_low: 20
+    target_temp_high: 25
+    keep_alive:
+      minutes: 3
+    initial_operation_mode: "auto"
+```
+
+Note: Make sure `target_temp_low` and `target_temp_high` are far enough from each other so you don't trigger both devices at the same time or bouce between.
```

Comment:
- Please use note boxes for this, see: https://home-assistant.io/developers/documentation/create_page/#html
- `bouce` -> `bounce`